bc_obps/compliance/schema/payments.py

```diff
@@ -0,0 +1,20 @@
+from typing import List
+from ninja import Schema
+
+
+class PaymentOut(Schema):
+    """Schema for a single payment record"""
+
+    id: str
+    paymentReceivedDate: str
+    paymentAmountApplied: float
```

Comment:
I'd suggest that for monetary values we use the Decimal type. I see there are lots of places where we're using 'float' across the module, I'll create a card to assess where we're using float & replace them with Decimal